src/cli/config-flags.ts

```diff
@@ -66,6 +66,7 @@ export const BOOLEAN_CLI_ARGS = [
   'notify',
   'onlyChanged',
   'onlyFailures',
+  'outputFile',
```

Comment:
This array that we're adding `outputFile` to is meant to be for [boolean arguments only](https://github.com/ionic-team/stencil/blob/bd1b8c288f0fa81145f9d67733f063539c082d6d/src/cli/config-flags.ts#L3-L6). For string arguments, we have a [separate array in this file](https://github.com/ionic-team/stencil/blob/bd1b8c288f0fa81145f9d67733f063539c082d6d/src/cli/config-flags.ts#L98-L101) that currently [contains `outputFile`](https://github.com/ionic-team/stencil/blob/bd1b8c288f0fa81145f9d67733f063539c082d6d/src/cli/config-flags.ts#L125).

In doing some research, `outputFile` is typed as [a string in Jest](https://github.com/facebook/jest/blob/f96bc2a6b6c23d3af76a9183bc37a09317d0fd1e/packages/jest-cli/src/cli/args.ts#L431-L435) and I'm not seeing it in `@types/jest`.
Is there a reason we need to add `outputFile` here?